packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -35,19 +35,19 @@ class EncryptionManager {
         }
     }
 
-    private shouldEncrypt(): boolean {
+    public shouldEncrypt(): boolean {
         return Boolean((this?.key as string) && (this.key as string).length > 0);
     }
 
-    private encrypt(str: string): [string, string | null, string | null] {
+    public encrypt(str: string): [string, string, string] {
         const iv = crypto.randomBytes(12);
         const cipher = crypto.createCipheriv(this.algo, Buffer.from(this.key!, this.encoding), iv);
         let enc = cipher.update(str, 'utf8', this.encoding);
         enc += cipher.final(this.encoding);
         return [enc, iv.toString(this.encoding), cipher.getAuthTag().toString(this.encoding)];
     }
 
-    private decrypt(enc: string, iv: string, authTag: string): string {
+    public decrypt(enc: string, iv: string, authTag: string): string {
```

Comment:
I am planning to refactor encryption manager so it only expose the `encrypt` and `decrypt` primitives and the logic to encrypt/decrypt environment/connection/records live in their respective package/service, like it is done in this PR